lib/db.js

```diff
@@ -261,6 +261,47 @@ class LimitDBRedis extends EventEmitter {
       });
   }
 
+  /**
+   * Get elements in the bucket.
+   *
+   * @param {getParams} params - The params for take.
+   * @param {function(Error, getResult)} [callback].
+   */
+  get(params, callback) {
+    callback = callback || _.noop;
+
+    const valError = this.validateParams(params, 'key');
+    if (valError) {
+      return process.nextTick(callback, valError);
+    }
+
+    const bucket = this.buckets[params.type];
+    const bucketKeyConfig = this.bucketKeyConfig(bucket, params.key);
+
+    if (bucketKeyConfig.unlimited) {
+      return process.nextTick(callback, null, {
+        remaining: bucketKeyConfig.size,
```

Comment:
I think that ‘size’ can be undefined when using unlimited:true. Might not be an issue, but just wanted to point out.